src/components/organisms/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -16,6 +16,7 @@ import {SELECT_LEFT_TOOL_PANEL, trackEvent} from '@utils/telemetry';
 
 import Colors from '@styles/Colors';
 
+import featureJson from '@src/feature-flags.json';
```

Comment:
let's remove the feature flag before we merge so that everyone on the team has the feature enabled by default for testing purposes